lib/common/tests/schemas/pcmk__cmp_schemas_by_name_test.c

```diff
@@ -74,18 +74,18 @@ known_numeric(void **state)
 }
 
 static void
-case_insensitive(void **state)
+case_sensitive(void **state)
```

Comment:
> real-world use cases

yeah should only matter for `cibadmin --empty` and `crm_shadow --create-empty`, if someone does a silly thing